Description: Fix HTTP proxy tests
Author: Jean Baptiste Favre <jbfavre@debian.org>
Origin: other
Forwarded: https://github.com/apache/trafficserver/pull/9885
Reviewed-by: Jean Baptiste Favre <jbfavre@debian.org>
Last-Update: 2022-06-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/proxy/http/Makefile.am
+++ b/proxy/http/Makefile.am
@@ -111,14 +111,16 @@ test_proxy_http_LDADD = \
 	$(top_builddir)/mgmt/libmgmt_p.la \
 	$(top_builddir)/iocore/utils/libinkutils.a \
 	@HWLOC_LIBS@ \
-	@LIBCAP@
+	@LIBCAP@ \
+	@YAMLCPP_LIBS@
 
 test_PreWarm_CPPFLAGS = \
 	$(AM_CPPFLAGS) \
 	-I$(abs_top_srcdir)/tests/include
 
 test_PreWarm_LDADD = \
-	$(top_builddir)/src/tscore/libtscore.la
+	$(top_builddir)/src/tscore/libtscore.la \
+	@YAMLCPP_LIBS@
 
 test_PreWarm_SOURCES = \
 	unit_tests/test_PreWarm.cc
@@ -158,6 +160,7 @@ test_HttpTransact_LDADD = \
 	@LIBZ@ \
 	@LIBLZMA@ \
 	@OPENSSL_LIBS@ \
+	@YAMLCPP_LIBS@ \
 	-lm
 
 test_HttpTransact_SOURCES = \
